packages/falcon-client/src/serviceWorker/sw.js

```diff
@@ -43,7 +43,9 @@ const router = new Router();
 self.addEventListener('fetch', event => {
   const responsePromise = router.handleRequest(event);
   if (responsePromise) {
-    event.respondWith(responsePromise);
+    // if response is redirected, we should clone the response before sending it
+    // see https://stackoverflow.com/a/45440505/412319
+    event.respondWith(responsePromise.then(res => (res.redirected ? res.clone() : res)));
```

Comment:
@nimobeeren why the previous `cleanResponse` function was removed?

```suggestion
    event.respondWith(responsePromise.then(res => (res.redirected ? cleanResponse(res) : res)));
```
I did a couple of tests, and the current implementation does not work on my side. I am receiving the same error as in reported issue.

Alo did som tests with suggested SO solution and it looks fine. Maybe we misunderstood in one of the previous comments?

Reply:
I also did some tests and it looks like @tutok is right - the solution with `cleanResponse(res)` works correctly for me too, while version from this PR doesn't so @nimobeeren please update it.